### DISCUSSION TITLE
Visibility of a user's authorizations

### DISCUSSION THREAD

**Discussion by phlpl** [zitadel/zitadel#5195] [Q&A] [answered]
I've been testing the application on the cloud and what is not clear to me if a user has visibility of their own authorizations.

I set up some projects, roles, applications, added a user, then assigned authorizations to that user.

When i log into the user account, there are 'no authorizations found' under authorizations.

Is the authorizations section under a user's profile only for those in a manager's role, where they can assign other user authorizations?  In that case, shouldn't these be hidden if they do not have a manager's role?

Is it possible for a user to view their authorizations and roles?

**Comment by hifabienne** [comment, marked as answer]
Hei @phlpl 
Thanks for bringing this up. I just had a look at what happens there and it looks like we are going to the management api (managing organizations) instead of the authentication api (managing/Read my own user).
If the user does not have permissions to manage the organizations it will return an error.  

I have created an issue according to the: https://github.com/zitadel/zitadel/issues/5199
